src/popper/runner_slurm.py

```diff
@@ -8,13 +8,18 @@
 from popper.cli import log as log
 from popper.runner_host import HostRunner
 from popper.runner_host import SingularityRunner as HostSingularityRunner
+from popper.utils import assert_executable_exists
 
 
 class SlurmRunner(HostRunner):
     def __init__(self, **kw):
         super(SlurmRunner, self).__init__(**kw)
         self._spawned_jobs = set()
 
+        slurm_executables = ["sbatch", "srun", "scancel", "mpirun"]
```

Comment:
for CI tests, these won't be available, and thus this fail as you correctly pointed out (on Slack). So one thing that can be done is to mock the `assert_executable_exists()` function in the test (`test_runner_slurm.py`).

Reply:
Gotcha! Thank you, I'm working on the mocking now.